devices/lepton/image14bit/color_test.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package image14bit
+
+import (
+	"testing"
+)
+
+func TestColor14(t *testing.T) {
+	if r, g, b, a := Color14(16383).RGBA(); r != 65532 || g != r || b != r || a != 65535 {
+		t.Fatal(r, g, b, a)
+	}
+	if r, g, b, a := Color14(0).RGBA(); r != 0 || g != r || b != r || a != 65535 {
+		t.Fatal(r, g, b, a)
+	}
+	if Color14(16383).String() != "Color14(16383)" || Color14(0).String() != "Color14(0)" {
+		t.Fail()
+	}
+	if Color14(8192) != convertColor14(Color14(8192)) {
+		t.Fail()
+	}
+}
```

Comment:
Please a test for convertColor14 code path lines 38-43, i.e. with a different color model.